src/components/Navigator/NavigatorCard.vue

```diff
@@ -194,10 +211,18 @@ export default {
       openNodes: {},
       /** @type {NavigatorFlatItem[]} */
       nodesToRender: [],
+      NO_RESULTS,
+      NO_CHILDREN,
+      ERROR_FETCHING,
+      ITEMS_FOUND,
     };
   },
   computed: {
     INDEX_ROOT_KEY: () => INDEX_ROOT_KEY,
+    itemsFoundNotification: ({ filteredChildren, children }) => {
+      const childrenLength = filteredChildren.length || children.length;
```

Comment:
If filtered children is 0, we should not fallback to the children. On SwiftUI that will say we have 40k items :D.

Use the `nodesToRender` instead, that would show only the items we are rendering atm.